subprojects/docs/src/snippets/testing/test-suite-configure-type/groovy/build.gradle

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+    id 'java'
+}
+
+version = '1.0.2'
+group = 'org.gradle.sample'
+
+repositories {
+    mavenCentral()
+}
+
+// tag::configure-type[]
+testing {
+    suites {
+        secondaryTest(JvmTestSuite) {
+            testType = TestSuitesType.INTEGRATION_TEST // <1>
```

Comment:
Should be `TestSuiteType` (singular), correct?

Can you verify this snippet is actually executing?

Reply:
It wasn't, fixed now.